bgpd/bgp_route.c

```diff
@@ -6717,10 +6746,17 @@ void route_vty_out(struct vty *vty, struct prefix *p, struct bgp_info *binfo,
 
 	/* Print aspath */
 	if (attr->aspath) {
-		if (json_paths)
+		if (json_paths) {
+
+			/*
+			 * Adding "path" field to match with corresponding
```

Comment:
I'm not sure I agree that path is better than aspath for this.

Reply:
@donaldsharp This is not about one is better than other in this case, but this is just to align the key words between CLI (Path) and Json (AsPath). I think both Path and AsPath are equally good here with respect to BGP. Agree ?